src/main/java/org/jenkinsci/test/acceptance/po/DumbSlave.java

```diff
@@ -42,20 +43,45 @@ public DumbSlave(Jenkins j, String name) {
      * call this in the context of the config UI.
      */
     public void asLocal() {
-        assertCurl();
-        File jar = new File("/tmp/slave"+createRandomName()+".jar");
-        String command = String.format(
-                "sh -c 'curl -s -o %1$s %2$sjnlpJars/slave.jar && java -jar %1$s'",
-                jar, url("../../")
-        );
+        assertCurlOrPowershellv3Plus();
+        File tmpDir = new File(System.getProperty("java.io.tmpdir"));
```

Comment:
Not convinced this should actually use tmp, but rather the build directory so mvn clean gets rid of the garbage but whatever.